go/libraries/doltcore/sqle/sqlselect_test.go

```diff
@@ -709,7 +709,7 @@ var BasicSelectTests = []SelectTest{
 				"0o2rnf5pq2s1nq3hj3609e1lt0socuf1",
 				"billy bob",
 				"bigbillieb@fake.horse",
-				time.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC),
+				time.Date(1970, 1, 1, 0, 0, 0, 0, &time.Location{}),
```

Comment:
Hmm, this might be the sign of a bug.

Reply:
So I I did a test by referencing the commit immediately before the `triggers 3` PR merge, and then the PR's commit, and this issue only popped up after the PR was merged. So it's definitely from that.